geopandas/geoseries.py

```diff
@@ -37,10 +37,10 @@ def _getitem_tuple(self, tup):
         if xs.step is not None or ys.step is not None:
             warn("Ignoring step - full interval is used.")
         xmin, ymin, xmax, ymax = obj.total_bounds
-        bbox = box(xs.start or xmin,
-                   ys.start or ymin,
-                   xs.stop or xmax,
-                   ys.stop or ymax)
+        bbox = box(xmin if xs.start is None else xs.start,
```

Comment:
Ha, I actually had that in the first place, and then convinced myself the `not None` was harder to parse than having the default value come last rather than first.